inc/functions/files.php

```diff
@@ -21,8 +21,8 @@ function get_rocket_advanced_cache_file() {
 
 	// Include the Mobile Detect class if we have to create a different caching file for mobile.
 	if ( is_rocket_generate_caching_mobile_files() ) {
-		$buffer .= "if ( file_exists( '" . WP_ROCKET_VENDORS_PATH . "classes/class-rocket-mobile-detect.php' ) && ! class_exists( 'Rocket_Mobile_Detect' ) ) {\n";
-		$buffer .= "\tinclude_once '" . WP_ROCKET_VENDORS_PATH . "classes/class-rocket-mobile-detect.php';\n";
+		$buffer .= "if ( file_exists( WP_CONTENT_DIR . '/plugins/wp-rocket/inc/vendors/classes/class-rocket-mobile-detect.php' ) && ! class_exists( 'Rocket_Mobile_Detect' ) ) {\n";
+		$buffer .= "\tinclude_once WP_CONTENT_DIR . '/plugins/wp-rocket/inc/vendors/classes/class-rocket-mobile-detect.php';\n";
```

Comment:
What's the reasoning behind changing those paths? The constant `WP_ROCKET_VENDORS_PATH` is perfectly fine here